nss-day-cohort-52/giffygram-commando-squad
# Message form Present

#### Changes Made
1. Added file `messageform.js` to `message` directory.
1. Modified file `message form` to include `the initial message form` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cthgiffy
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When user is logged in they should be able to see the message form on the screen. currently the send/cancel buttons are not functioning for the confirmation message but they are present on the page.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
